### ISSUE TITLE
[$15]{MSFT-120} Challenges page: Screen reader doesn't announce the type of element for the 'compte ' and 'community' controls.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2818]
**Steps to Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 2. Ensure screen reader is on. 3. Press down arrow to read the 'compte' and 'community' controls. 3. Notice that the screen reader doesn't announce these as either buttons or links.

**Expected Result**
Compete and community controls should be announced as links since they move focus to a different page.

**Actual Result**
Screen reader doesn't announce the type of element for the 'compete' and 'community' control.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 4. Robust
**WCAG Level**: A
**WCAG Success Criterion**: 4.1.2 Name, Role, Value
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2031

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096042 has been created for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096042 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096042 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by codeMinter** [comment]
@veshu this is already fixed as part of #2754

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097276 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097276 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097276 has been updated - it has been assigned to gets0ul.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097276<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@crazyk07 I shall be paid for this as my PR was merged. @jmgasper assigned getsoul, whereas I also referenced this back long.

can this be assigned to me now

**Comment by PrakashDurlabhji** [comment]
@crazyk07 this prize should be given to me, I even referenced this earlier. request you to take necessary action. waiting here

**Comment by PrakashDurlabhji** [comment]
@crazyk07  I know lots of ticket are to be processed but I dont wish to lose efforts.

**Comment by PrakashDurlabhji** [comment]
@jmgasper

**Comment by PrakashDurlabhji** [comment]
@veshu can you please help with above urgently, as I guess other members havent noticed it due to lots of ticket

**Comment by veshu** [comment]
@PrakashDurlabhji Thanks for bringing this issue, i went through the activity on this issue and found that the gets0ul has already assigned and submitted PR before you have referenced it.
![image](https://user-images.githubusercontent.com/2202045/62219743-953dc900-b3ce-11e9-88c0-63b403be4e9b.png)

And the solution he has provided is also valid not merged just because it was already fixed. 
Hope you understand the situation.

**Comment by PrakashDurlabhji** [comment]
@veshu let me show you a comment from another ticket which i did that time itself. just 5 mins

**Comment by PrakashDurlabhji** [comment]
@veshu  kindly check comment of codeMinter as shown in image which gives link to issue #2754 .

1. Kindly go to #2754 which was solved by me.
2. and PR for #2754  was raised long back.
3. you can check my comment which is on issue 2754 and link for that is: https://github.com/topcoder-platform/community-app/issues/2754#issuecomment-513299964

4. Also the solution which codeminter mentioned about is mine solution. you can verify from above links.

Thanks waiting for reply.

**Comment by veshu** [comment]
@PrakashDurlabhji Yes i went on those, please check carefully in the screenshot i have shared which is from this issue's activity, you have referenced this issue after one day, we got the PR from gets0ul.

**Comment by PrakashDurlabhji** [comment]
@veshu kindly check my this comment https://github.com/topcoder-platform/community-app/issues/2754#issuecomment-512944129 which was 13 days ago and 1st commit of that PR is 13 days ago.

I was late to comment by 1 day hence I even mentioned detailed scenario here https://github.com/topcoder-platform/community-app/issues/2754#issuecomment-513299964 that review my PR first as i always aware this issue belongs to me. 

and getsOul PR was never merged because I solved it and same was commented by codeMinter as shown in image.

Kindly request you to see timeline of above comments link and commits link.

@gets0ul can you please truly say whether this issue truely belongs to you or due to your PR merging?

**Comment by veshu** [comment]
@PrakashDurlabhji Don't take offended, but as I mentioned above, even I showed you the activity timeline, where it is clear that you have referenced the issue later.

May be you are confused with multiple commits you have made on that PR. You have created 3 commits among which one is created 13 days ago. but after that you updated the PR after gets0ul committed the PR.

Check the detail screen shot from both of you for the same changes `role=link`

https://prnt.sc/omjwdo (Your commit)
https://prnt.sc/omjwx6 (gets0ul's commit)

gets0ul has committed at 2019-7-18T23:21:18z, where as your commit is at 2019-7-19T08:42:22z

Hope you are clear now. Thanks!

**Comment by PrakashDurlabhji** [comment]
@veshu I am not offended, no problem I completely understand just as his PR was not merged hence I wanted to discuss. Yes I am clear now.

**Comment by crazyk07** [comment]
@PrakashDurlabhji  Thanks for understanding.  Sorry, just catching up with this.  Thanks for your participation thus far!
